L1Trigger/L1THGCal/test/testHGCalL1T_cfg.py

```diff
@@ -25,7 +25,7 @@
 
```

Comment:
Keep this file as it was before, except for what needs to be added for the calibration.